### PR TITLE
trino 407

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  trino:
    * Formula refers to a GitHub issue or pull request that is closed: https://github.com/trinodb/trino/issues/8877
```

---

relates to https://github.com/Homebrew/brew/pull/14326

File: Formula/trino.rb [hunk 1]
```diff
@@ -21,25 +21,23 @@ class Trino < Formula
...
     # Manually extract tarball to avoid losing hardlinks which increases bottle
     # size from MBs to GBs. Remove once Homebrew is able to preserve hardlinks.
-    # Ref: https://github.com/Homebrew/brew/pull/13154
     libexec.mkpath
     system "tar", "-C", libexec.to_s, "--strip-components", "1", "-xzf", "trino-server-#{version}.tar.gz"
 
     # Manually untar, since macOS-bundled tar produces the error:
     #   trino-363/plugin/trino-hive/src/test/resources/<truncated>.snappy.orc.crc: Failed to restore metadata
-    # Remove when https://github.com/trinodb/trino/issues/8877 is fixed
     resource("trino-src").stage do |r|
```
Comment (SMillerDev) [inline Formula/trino.rb:41]: If upstream thinks it's fixed, why do we still do this?

File: Formula/trino.rb [hunk 2]
```diff
@@ -21,25 +21,23 @@ class Trino < Formula
...
-    url "https://search.maven.org/remotecontent?filepath=io/trino/trino-cli/406/trino-cli-406-executable.jar"
-    sha256 "70f7ab57cf74f655942d14f4baafebac32ca64ca833790411691a69b574c16bb"
+    url "https://search.maven.org/remotecontent?filepath=io/trino/trino-cli/407/trino-cli-407-executable.jar"
+    sha256 "1b1fd386501758b7a392a4791e49da061614b2890d3bf76ecfa130aa56380912"
   end
 
   def install
     # Manually extract tarball to avoid losing hardlinks which increases bottle
     # size from MBs to GBs. Remove once Homebrew is able to preserve hardlinks.
-    # Ref: https://github.com/Homebrew/brew/pull/13154
```
Comment (SMillerDev) [inline Formula/trino.rb:36]: This should be made into an issue then

File: Formula/trino.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/trino.rb:41]: the issue is actually fixed, but got closed. https://github.com/trinodb/trino/issues/8877 (I guess we just did not do the followup)

Comment (chenrui333) [reply Formula/trino.rb:41]: Actually this does work for me now, let me update the PR

```
$ curl -sSL https://github.com/trinodb/trino/archive/360.tar.gz | tar -xz
$
```

File: Formula/trino.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/trino.rb:36]: yeah, that sounds about right

File: Formula/trino.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/trino.rb:41]: actually still an issue. The zip file does not work either

**Comment by chenrui333** [comment]
I think for now we can just merge as it is.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**